sycl/test-e2e/Matrix/Inputs/joint_matrix_bfloat16_PackedB_impl.hpp

```diff
@@ -0,0 +1,195 @@
+//===---joint_matrix_bfloat16_impl.hpp - DPC++ joint_matrix----------------===//
```

Comment:
Fix the name here so it does not affect 
Matrix/SG32/joint_matrix_bfloat16_packedB.cpp